subtitles.go

```diff
@@ -189,6 +239,8 @@ type StyleAttributes struct {
 	STLBoxing            *bool
 	STLItalics           *bool
 	STLUnderline         *bool
+	STLVerticalPostion   int
```

Comment:
Can you create a specific type `stlVerticalPosition` for that and make it a pointer in `StyleAttributes`?

`type stlVerticalPosition int`